examples/lighting-app/cyw30739/src/ButtonHandler.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * $ Copyright 2021-YEAR Cypress Semiconductor $
```

Comment:
Please fix copyright throughout.

Also the comment of `@app_button.cpp` seems not useful - looks like a copy and paste of a different name. Please delete.